test/test_nn.py

```diff
@@ -10055,7 +10055,7 @@ def test_Dropout(self, device):
         input = torch.Tensor(1000)
```

Comment:
how the hell does this work? It's an uninitialized tensor!
Also, looks like dropout is not tested for datatypes other than default (double?float?) and bfloat16 ? Seems dangerous.

Reply:
fixed